app/questionnaire/path_finder.py

```diff
@@ -16,6 +15,8 @@ def __init__(self, survey_json, answer_store=None, metadata=None):
         self.answer_store = answer_store or AnswerStore()
         self.metadata = metadata or {}
         self.survey_json = survey_json
+        self.answer_store_hash = self.answer_store.get_hash()
```

Comment:
should these be `_` members?

Reply:
@collisdigital I think you're right, `.answer_store_hash` and `.full_routing_path` should be as they are only to be used and calculated by the instance's methods.